build.sbt

```diff
@@ -1,6 +1,4 @@
-scalaVersion := "2.10.6"
-
-crossScalaVersions := Seq("2.10.6", "2.11.8")
+scalaVersion in ThisBuild := "2.11.8"
```

Comment:
I've checked libraries which tranquility uses and it's not trivial to add scala 2.12 support right now (for example scala 2.12 support in flink is coming soon, spark version should be updated also and so on) . I would suggest dropping scala 2.10 support for now and then add 2.12 support later by another PR.